scipy/linalg/_solvers.py

```diff
@@ -158,7 +158,7 @@ def solve_discrete_lyapunov(a, q, method=None):
         A square matrix
 
     q : (M, M) array_like
```

Comment:
By the way, shouldn't Q be a symmetric (Hermitian) at least? This is worth including to the description.